packages/react/src/hooks/useCombinedRef.ts

```diff
@@ -0,0 +1,31 @@
+import { Ref, useCallback, MutableRefObject, RefCallback } from 'react';
+
+/**
+ * 여러 개의 ref를 합칠 수 있는 훅입니다.
+ * @example
+ * const Foo = forwardRef((props, fowardedRef) => {
+ *   const ref = useRef();
+ *   const combinedRef = useCombinedRefs(fowardedRef, ref2);
+ *
+ *   // div가 업데이트되면 ref, fowardedRef 둘 다 업데이트 됨
+ *   return <div ref={combinedRef} />
+ * });
+ */
+export default function useCombinedRefs<T>(
+  ...refs: Array<Ref<T> | RefCallback<T>>
```

Comment:
```suggestion
  ...refs: Array<Ref<T>>
```
<br />

`Ref<T>`에 `RefCallback<T>`도 포함되어있어서 이렇게만 적어줘도 괜찮을 것 같아요..!

```typescript
type Ref<T> = RefCallback<T> | RefObject<T> | null;
```

Reply:
https://github.com/Lubycon/lubycon-frontend-libraries/pull/44/commits/9b25ed7c4b65d779c74f3936946859b2a41a0f0b 에서 반영되었습니다!